odh-dashboard/base/kustomization.yaml

```diff
@@ -5,8 +5,17 @@ commonLabels:
   app.kubernetes.io/part-of: odh-dashboard
 resources:
 - role.yaml
+- cluster-role.yaml
 - service-account.yaml
 - role-binding.yaml
+- cluster-role-binding.yaml
 - deployment.yaml
 - routes.yaml
 - service.yaml
+- odhapplicationcrd.yaml
+- odhdocumentcrd.yaml
+- odhquickstartcrd.yaml
+- applications/jupyterhub.yaml
+- docs/jupyterhub-docs.yaml
+- quickstarts/create-jupyter-notebook-quickstart.yaml
+- quickstarts/deploy-python-model-quickstart.yaml
```

Comment:
Looks like there is an issue adding the CRs right after adding the CRDs:
```
unable to recognize ".": no matches for kind "OdhApplication" in version "applications.console.openshift.io/v1alpha1"
unable to recognize ".": no matches for kind "OdhQuickStart" in version "console.openshift.io/v1"
unable to recognize ".": no matches for kind "OdhQuickStart" in version "console.openshift.io/v1"
unable to recognize ".": no matches for kind "OdhDocument" in version "documents.console.openshift.io/v1alpha1"
unable to recognize ".": no matches for kind "OdhDocument" in version "documents.console.openshift.io/v1alpha1"
unable to recognize ".": no matches for kind "OdhDocument" in version "documents.console.openshift.io/v1alpha1"
unable to recognize ".": no matches for kind "OdhDocument" in version "documents.console.openshift.io/v1alpha1"
```

@crobby @cfchase Is there some other place to add the jupyterhub CRs that would be after the CRDs have been established?

Reply:
Maybe if they were in another component that comes after odh-dashboard in the KfDef, they would get created after the CRDs?

Reply:
Since kfdef does not support staggered installs, I think we only have two options
* Use an OpenShift job to apply these after a delay but this would probably complicate the reconcile of these objects
* Move the CRDs to odh-common --OR-- relocate these objects to the ODH operator bundle so that they are added when the operator is installed.

Reply:
It might make more sense to move the CRDs to `odh-common` then add the jupyterhub resources to `jupyterhub`. Thoughts @LaVLaS?